src/context.ts

```diff
@@ -19,16 +19,22 @@ export class CommandContext {
    * Alias for `this.context.sourceCode`
    */
   readonly source: Linter.SourceCode
+  /**
+   * Captured groups by match
+   */
+  readonly groups: string[]
```

Comment:
Maybe instead of groups, we could directly expose the `match: RegExpMatchArray`? There would also be cases like named groups etc.